spec/appwindow-spec.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
+
+var assert = require('assert');
+var sinon = require('sinon');
+var SandboxedModule = require('sandboxed-module');
+var remote = require('remote');
+
+describe('AppWindow', function () {
+
+  beforeEach(function () {
+    var self = this;
```

Comment:
Just a habit. Uglify doesn't minify `this` keyword. And we are not using it here it so makes sense. Thanks for pointing out. Fixed it :+1: